### DISCUSSION TITLE
question about map data

### DISCUSSION THREAD

**Discussion by farfromrefug** [andreynovikov/trekarta#190] [General]
Hi i am a fellow developer and i also do my own hiking/map app https://github.com/Akylas/alpimaps (though right now only for personal use).
I really like what you have done!
I have questions about the mtiles you use in your app:
* how do you generate that data? do you use openmaptiles?
* is that publicly available?
* what does the processing do on the downloaded files?

I think we can help each other in making our apps better!
Right now my use is generating all my data from openmaptiles (mbtiles, hillshades, contours) for my whole country, store them on my sdcard and the app read that automatically.
I want to mention to you that i use Carto SDk(https://github.com/CartoDB/mobile-sdk) as map framework. I think it can do most of what you do, fully OSS. And to be honest perfs are much better than oscim. With your app if i download a region with  hillshade i simply cant move the map anymore, it is too slow. With carto SDK it works almost flawlessly

Hope to hear from you,
Thanks for your great work!
Martin

**Comment by andreynovikov** [comment]
Yes, I preprocess OSM data with https://github.com/andreynovikov/mapcreator
